site/docs/pages/index.mdx

```diff
@@ -84,14 +84,14 @@ bun add @coinbase/onchainkit
       <div>
 
 ```tsx
-<OnchainKitProvider
-  chain={base}
-  schemaId="0xf8b05c79f090979bf4a80270aba232dff11a10d9ca55c4f88de95317970f0de9"
->
+<OnchainKitProvider // [!code focus]
```

Comment:
Open question, why you think before we didn't have `code focus` here?

Reply:
I assumed we need to highlight any onchainKit components. Should I remove it here?

Reply:
So let me ask you a question, why you think it was not highlighted before?